default/python/AI/AIDependencies.py

```diff
@@ -62,8 +61,16 @@
 SHIP_WEAPON_DAMAGE_FACTOR = fo.getGameRules().getDouble("RULE_SHIP_WEAPON_DAMAGE_FACTOR")
 FIGHTER_DAMAGE_FACTOR = fo.getGameRules().getDouble("RULE_FIGHTER_DAMAGE_FACTOR")
 
-# Colonization details
-OUTPOST_POD_COST = 50  # TODO Query directly from part
+
+def outpost_pod_cost():
+    """
+    Get current outpost cost.
+
+    This value is changed on each turn, don't cache it.
+    """
+    return fo.getShipPart("CO_OUTPOST_POD").productionCost(fo.empireID(), -1, -1)
```

Comment:
I think it would better to use PlanetUtilsAI.get_capital() as location. At the moment the location does not matter, but if the scripting changes so that it does, the function would return ARBITRARY_LARGE_COST instead of a cost that is correct at least at one likely candidate.

Function must moved for that, but you can just as well change the existing function outpod_pod_cost in ColonisationAI, it seems unlikely that any other part ever needs this value.

Also the value could be cached with cache_for_current_turn, but it may not be worth the effort.